### DISCUSSION TITLE
ERROR on Podman

### DISCUSSION THREAD

**Discussion by KryCool** [containers/podman#23069] [Q&A]
The error when I use command podman pull node:20

- Error: creating build container: initializing image from source docker://localhost:9002/node:20: rejecting ambiguous manifest, unexpected fields []string{"manifests"} in supposedly application/vnd.docker.distribution.manifest.v2+json

When I'm checking the podman image inspect node the error to show 
Error: parsing manifest blob "{\"manifests\":[{\"annotations\":{\"com.docker.official-images.bashbrew.arch\":\"amd64\",\"org.opencontainers.image.base.digest\":\"sha256:9cb4155468e01fe11d28e9e878a509934e3f26ae828136ce5fb6b154dc1978df\",\"org.opencontainers.image.base.name\":\"buildpack-deps:bookworm\",\"org.opencontainers.image.created\":\"2024-06-21T01:02:44Z\",\"org.opencontainers.image.revision\":\"daea62837e99456d7556b585edbc2b32fb57369e\",\"org.opencontainers.image.source\":\"https:\\/\\/github.com\\/nodejs\\/docker-node.git#daea62837e99456d7556b585edbc2b32fb57369e:20\\/bookworm\",\"org.opencontainers.image.url\":\"https:\\/\\/hub.docker.com\\/_\\/node\",\"org.opencontainers.image.version\":\"20\"},\"digest\":\"sha256:23d8c295bddde37ad2ca6bfc81fb236aec83ee2a919411aeb520fcde195f98fe\",\"mediaType\":\"application\\/vnd.oci.image.manifest.v1+json\",\"platform\":{\"architecture\":\"amd64\",\"os\":\"linux\"},\"size\":2493},{\"annotations\":{\"com.docker.official-images.bashbrew.arch\":\"amd64\",\"vnd.docker.reference.digest\":\"sha256:23d8c295bddde37ad2ca6bfc81fb236aec83ee2a919411aeb520fcde195f98fe\",\"vnd.docker.reference.type\":\"attestation-manifest\"},\"digest\":\"sha256:2de177aa1f41d13db9324140c18ecf80b90c27f1f82c6610bafd63a2221d12af\",\"mediaType\":\"application\\/vnd.oci.image.manifest.v1+json\",\"platform\":{\"architecture\":\"unknown\",\"os\":\"unknown\"},\"size\":842},{\"digest\":\"sha256:07270d34f7838b7aa89f252ccc6e22afc87d8e533c469f0287d4f9b5f23760c8\",\"mediaType\":\"application\\/vnd.docker.distribution.manifest.v2+json\",\"platform\":{\"architecture\":\"arm\",\"os\":\"linux\",\"variant\":\"v7\"},\"size\":2004},{\"digest\":\"sha256:a30a991ea562ccaab2f43692acae70d34d63f2361e36c35bd023b527f75168b9\",\"mediaType\":\"application\\/vnd.docker.distribution.manifest.v2+json\",\"platform\":{\"architecture\":\"arm64\",\"os\":\"linux\",\"variant\":\"v8\"},\"size\":2004},{\"annotations\":{\"com.docker.official-images.bashbrew.arch\":\"ppc64le\",\"org.opencontainers.image.base.digest\":\"sha256:3fe609d9d847bf31a6147a461634bef76e5e5acd6ca7b260c4042517244b84f0\",\"org.opencontainers.image.base.name\":\"buildpack-deps:bookworm\",\"org.opencontainers.image.created\":\"2024-06-21T04:03:18Z\",\"org.opencontainers.image.revision\":\"daea62837e99456d7556b585edbc2b32fb57369e\",\"org.opencontainers.image.source\":\"https:\\/\\/github.com\\/nodejs\\/docker-node.git#daea62837e99456d7556b585edbc2b32fb57369e:20\\/bookworm\",\"org.opencontainers.image.url\":\"https:\\/\\/hub.docker.com\\/_\\/node\",\"org.opencontainers.image.version\":\"20\"},\"digest\":\"sha256:a5afa4b7de047a7cf5df95e7f7d39b561594790e7e86066759eb9b1d7961c7d0\",\"mediaType\":\"application\\/vnd.oci.image.manifest.v1+json\",\"platform\":{\"architecture\":\"ppc64le\",\"os\":\"linux\"},\"size\":2495},{\"annotations\":{\"com.docker.official-images.bashbrew.arch\":\"ppc64le\",\"vnd.docker.reference.digest\":\"sha256:a5afa4b7de047a7cf5df95e7f7d39b561594790e7e86066759eb9b1d7961c7d0\",\"vnd.docker.reference.type\":\"attestation-manifest\"},\"digest\":\"sha256:91ae8f164616c6abdfb33c1e7992d679300dc7c19f42e19824008567bfde10cd\",\"mediaType\":\"application\\/vnd.oci.image.manifest.v1+json\",\"platform\":{\"architecture\":\"unknown\",\"os\":\"unknown\"},\"size\":842},{\"digest\":\"sha256:4eb5dc0f5c419787f9e61e63c0cc74bb07c58c688b362d0d66192378c210a640\",\"mediaType\":\"application\\/vnd.docker.distribution.manifest.v2+json\",\"platform\":{\"architecture\":\"s390x\",\"os\":\"linux\"},\"size\":2004}],\"mediaType\":\"application\\/vnd.oci.image.index.v1+json\",\"schemaVersion\":2}" as a "application/vnd.docker.distribution.manifest.v2+json": rejecting ambiguous manifest, unexpected fields []string{"manifests"} in supposedly application/vnd.docker.distribution.manifest.v2+json

I have checked on Google for many hours, but nothing to help.
Thanks.

**Comment by nalind** [comment]
That manifest includes items with both OCI (the amd64 and ppc64le images, and their attestations) and Docker (the arm/arm64/s390x images) manifest types.  Where did it come from?
